ui/components/component-library/banner-base/banner-base.types.ts

```diff
@@ -31,7 +31,7 @@ export interface BannerBaseStyleUtilityProps extends StyleUtilityProps {
   /**
    * The description is the content area below BannerBase title
    */
-  description?: string;
+  description?: string | React.ReactNode;
```

Comment:
The description is wrapped by `p` tag what are we passing here that requires a node? The banners should also accept a `children` prop which might be more suited for a custom node.